.tsqllint-version

```diff
@@ -0,0 +1 @@
+1.11.1
```

Comment:
rename to `tsqllint.version` no need to be hidden

Reply:
Done